llvm/test/CodeGen/AVR/jmp.ll

```diff
@@ -0,0 +1,24 @@
+; RUN: llc -filetype=obj -mtriple=avr < %s | llvm-objdump -d --no-show-raw-insn - | FileCheck %s
```

Comment:
```suggestion
; RUN: llc -filetype=obj -mtriple=avr < %s | llvm-objdump -dr --no-show-raw-insn - | FileCheck %s
```

It might be a good idea to verify that no relocation is generated, so that the test really tests for the current behavior.